ssc_interface_wrapper_ros2/launch/remapped_ssc_interface.launch.py

```diff
@@ -35,8 +36,27 @@ def generate_launch_description():
     command_timeout = LaunchConfiguration('command_timeout')
     declare_command_timeout = DeclareLaunchArgument(name ='command_timeout', default_value='1000')
 
-    loop_rate = LaunchConfiguration('loop_rate')
-    declare_loop_rate = DeclareLaunchArgument(name ='loop_rate', default_value='30.0')
+    status_pub_rate = LaunchConfiguration('status_pub_rate')
+    declare_status_pub_rate = DeclareLaunchArgument(name ='status_pub_rate', default_value='30.0')
+
+    wheel_base = LaunchConfiguration('wheel_base')
+    declare_wheel_base = DeclareLaunchArgument(name='wheel_base', default_value='2.79')
+
+    tire_radius = LaunchConfiguration('tire_radius')
+    declare_tire_base = DeclareLaunchArgument(name='tire_radius', default_value='0.39')
+
+    acceleration_limit = LaunchConfiguration('acceleration_limit')
+    declare_acceleration_limit =DeclareLaunchArgument(name='acceleration_limit', default_value='3.0')
+    
+    deceleration_limit = LaunchConfiguration('deceleration_limit')
+    declare_deceleration_limit =DeclareLaunchArgument(name='deceleration_limit', default_value='3.0')
+
+    max_curvature_rate = LaunchConfiguration('max_curvature_rate')
+    declare_max_curvature_rate = DeclareLaunchArgument(name='max_curvature_rate', default_value='0.15')
```

Comment:
How are these being set via the launch arguments? 
Having a launch argument doesn't mean a ros parameter exists. You would still need to pass them into the node. But then that begs the question how these would get set from the parameter file. 
Following the pattern used in the other ros2 files pass in the config file path and load that as an additional parameter file in your node launch.